db/migrate/20220311003928_create_branches.rb

```diff
@@ -5,7 +5,7 @@ def change
       t.string :city
       t.string :town
       t.integer :number
-      t.boolean :status
+      t.boolean :status, default: true
```

Comment:
this is good because it adds a default value that probably you needed and solves a problem, but it introduces a little thing that you are not seeing ... and this is a good example to learn from it:

from the migrations, what are migrations?

> Migrations are a feature of Active Record that allows you to evolve your database schema over time. Rather than write schema modifications in pure SQL, migrations allow you to use a Ruby DSL to describe changes to your tables.
>
> https://guides.rubyonrails.org/active_record_migrations.html

and in this case this migration `db/migrate/20220311003928_create_branches.rb` was created on 2022-03-11 like 3 weeks ago ... How do I know this? by looking at the name of the migration `20220311003928` this value is the date of creation of this migration.

And at the time, 3 weeks ago you said: 
- create this table :branches with this fields.

And now, today 3 weeks later you want to say:
- Add a default value for specific field in table :branches

In this case it is happening 3 weeks later So it is a completely different change, and a completely different migration, so you need to create a new one for this specific change you are proposing.

what I am trying to explain is also explained here with different words https://guides.rubyonrails.org/active_record_migrations.html#changing-existing-migrations